src/javascript/app/pages/trade/charts/highchart.js

```diff
@@ -363,37 +369,52 @@ const Highchart = (() => {
+                    if (Reset.isNewBarrier(entry_spot, barrier)) {
+                        addPlotLine({ id: 'barrier',       value: entry_spot * 1, label: localize('Barrier ([_1])', [addComma(entry_spot)]),     dashStyle: 'Dot',   textBottom: contract_type !== 'RESETCALL', x: -60, align: 'right'  }, 'y');
+                        addPlotLine({ id: 'reset_barrier', value: barrier * 1,    label: localize('Reset Barrier ([_1])', [addComma(barrier)]),  dashStyle: 'Solid', textBottom: contract_type === 'RESETCALL', x: -60, align: 'right'   }, 'y');
+                    } else {
+                        addPlotLine({ id: 'barrier',       value: entry_spot * 1, label: localize('Barrier ([_1])', [addComma(entry_spot)]),     dashStyle: 'Dot', x: -60, align: 'right'  }, 'y');
+
+                    }
                 } else {
-                    addPlotLine({ id: 'barrier',      value: barrier * 1,      label: localize('Barrier ([_1])', [addComma(barrier)]),           dashStyle: 'Dot' }, 'y');
+                    addPlotLine({ id: 'barrier',       value: barrier * 1,      label: localize('Barrier ([_1])', [addComma(barrier)]),            dashStyle: 'Dot' },   'y');
                 }
             } else if (high_barrier && low_barrier) {
                 prev_barriers[1] = high_barrier;
                 prev_barriers[0] = low_barrier;
                 if (Lookback.isLookback(contract_type)) {
                     const [high_label, low_label] = Lookback.getBarrierLabel(contract_type);
-                    addPlotLine({ id: 'high_barrier', value: high_barrier * 1, label: localize(`${high_label} ([_1])`, [addComma(high_barrier)]), dashStyle: 'Dot' }, 'y');
-                    addPlotLine({ id: 'low_barrier',  value: low_barrier * 1,  label: localize(`${low_label} ([_1])`, [addComma(low_barrier)]),   dashStyle: 'Dot', textBottom: true }, 'y');
+                    addPlotLine({ id: 'high_barrier',  value: high_barrier * 1, label: localize(`${high_label} ([_1])`, [addComma(high_barrier)]), dashStyle: 'Dot' }, 'y');
+                    addPlotLine({ id: 'low_barrier',   value: low_barrier * 1,  label: localize(`${low_label} ([_1])`, [addComma(low_barrier)]),   dashStyle: 'Dot', textBottom: true }, 'y');
                 } else {
-                    addPlotLine({ id: 'high_barrier', value: high_barrier * 1, label: localize('High Barrier ([_1])', [addComma(high_barrier)]), dashStyle: 'Dot' }, 'y');
-                    addPlotLine({ id: 'low_barrier',  value: low_barrier * 1,  label: localize('Low Barrier ([_1])', [addComma(low_barrier)]),   dashStyle: 'Dot', textBottom: true }, 'y');
+                    addPlotLine({ id: 'high_barrier',  value: high_barrier * 1, label: localize('High Barrier ([_1])', [addComma(high_barrier)]),  dashStyle: 'Dot' }, 'y');
+                    addPlotLine({ id: 'low_barrier',   value: low_barrier * 1,  label: localize('Low Barrier ([_1])', [addComma(low_barrier)]),    dashStyle: 'Dot', textBottom: true }, 'y');
```

Comment:
can we revert the changes in these four lines (405, 406, 408, 409) since it's all whitespace and not needed anymore?